documentation/source/base/design_build.rst

```diff
@@ -8,33 +8,33 @@ The two main classes are:
 * :py:class:`bluemira.base.designer.Designer`: the Designer base class
 * :py:class:`bluemira.base.builder.Builder`: the Builder base class
 
-A :py:class:`Designer` carries out component design activities,
+A :py:class:`~bluemira.base.designer.Designer` carries out component design activities,
 for example evaluating a geometry parameterisation,
 or performing an optimisation.
 It outputs a minimal representation of a component that can be used to generate geometry
 (note that this minimal representation may itself be some, simple, geometry).
 
-A :py:class:`Builder` is responsible for generating and manipulating ``bluemira``
-geometry objects (CAD), to create a :py:class:`Component`.
+A :py:class:`~bluemira.base.builder.Builder` is responsible for generating and manipulating ``bluemira``
+geometry objects (CAD), to create a :py:class:`~bluemira.base.components.Component`.
 It will typically perform geometry operations like offsets or sweeps,
 and is not intended to perform any complex calculations.
 
 Designers
 ^^^^^^^^^
 
-:py:class:`Designers` solve the minimal design problem of a :py:class:`Component`.
+:py:class:`~bluemira.base.designer.Designer`\s solve the minimal design problem of a :py:class:`~bluemira.base.components.Component`.
```

Comment:
It might be a good to also say something like "...of a Component or any other object that requires calculation or optimisation"

Because equilibria for ex. don't have a component really and they are a big part of "designing"